server/routes/config/additionalConditionsV1.ts

```diff
@@ -0,0 +1,332 @@
+export const additionalConditionsV1 = {
```

Comment:
extracted out separate modules for validation messages for each set of conditions